src/main/java/mate/academy/hw02/sort/BuubleSort.java

```diff
@@ -0,0 +1,66 @@
+package mate.academy.hw02.sort;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class BuubleSort {
+    public static void main(String[] args) {
+
+        long t1 = System.nanoTime();
+
+        int counter, num, array[];
+
+        //Создаем объект Scanner для считывания чисел, введенных пользователем
+        Scanner input = new Scanner(System.in);
+        System.out.println("Введите количество элементов массива: ");
+        num = input.nextInt();
+
+        // Создаем массив введенного пользователем размера
+        array = new int[num];
+
+        System.out.println("Введите " + num + " чисел");
+
+        //Заполняем массив, вводя элементы в консоль
+        for (counter = 0; counter < num; counter++) {
+            array[counter] = input.nextInt();
+        }
+
+        // печатаем массив перед пузырьковой сортировкой
+        System.out.println("массив перед пузырьковой сортировкой : " + Arrays.toString(array));
+
+        // сортируем массив
+        bubbleSort(array);
+
+        // печатаем массив после пузырьковой сортировки
+        System.out.println("массив после пузырьковой сортировки : " + Arrays.toString(array));
+
+        System.out.println("Время выполнения программы:  " + (System.nanoTime() - t1) / 1_000_000);
+    }
+
+    // метод пузырьковой сортировки
+    public static void bubbleSort(int[] num) {
+
+        int j;
+        boolean flag = true;   // устанавливаем наш флаг в true для первого прохода по массиву
```

Comment:
remove comments,code should be self descriptive

Reply:
I've changed